packages/fxa-auth-server/scripts/clean-up-old-ci-stripe-customers.js

```diff
@@ -65,6 +65,9 @@ async function init() {
   const config = require('../config').getProperties();
   const log = require('../lib/log')(config.log.level);
 
+  Container.set(AppConfig, config);
+  Container.set(AuthLogger, log);
```

Comment:
Glad you saw the comment in the Jira about how the GoogleMapsService assumes these two objects are already set on the container. Do we expect that service to be used in these various scripts?

Reply:
It doesn't matter if the geocoding feature is used in the scripts or not.  GoogleMapsService is now a dependency in StripeHelper, and those are GoogleMapsService's dependencies.

Reply:
Ah so anywhere that uses StripeHelper also needs to set those objects in the container. I got it thanks for clarifying.